packages/x-charts-pro/package.json

```
@@ -53,8 +53,8 @@
   "peerDependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
-    "@mui/material": "^5.15.14 || ^6.0.0 || ^7.0.0-alpha",
-    "@mui/system": "^5.15.14 || ^6.0.0 || ^7.0.0-alpha",
+    "@mui/material": "^5.15.14 || ^6.0.0 || ^7.0.0-beta",
```

I would already define the full v7 range so that we don't have to make changes to test with stable core once it gets release. We then just have to remove `^7.0.0-beta` once X goes stable.

```suggestion
    "@mui/material": "^5.15.14 || ^6.0.0 || ^7.0.0 || ^7.0.0-beta",
```

---

Whoops, somehow I might have missed something. 🙈 
Clearly, I agree with the proposal. 👌

---

And I'm just still wondering about the devdependencies. I feel like we should think about how this will look in stable v7 world. i.e. every time we'd write

```json
"@mui/material": "^7.3.8"
```

we'd want that to look like

```json
"@mui/material": "^7.0.0"
```

At the time of X@v8 going stable. And thus today, in beta, we want that to look like

```json
"@mui/material": "^7.0.0 || ^7.0.0-beta"
```
or

```json
"@mui/material": "^7.0.0 || ^7.0.0-beta.4"
```

---

Does it matter that much if we plan to remove the `beta` dep after a stable release?
Technically, we could have probably left `^7.0.0-alpha.0` to be precise, if there are no specific fixes afterwards, that would prevent interoperability. 🤔

---

It's not as important as it's not user facing. We can update it as well when core@v7 becomes stable.